README.md

```diff
@@ -8,26 +8,31 @@ This repository hosts workflows to process several data sources and cleaned data
-- [`output/cases/cases_us_states_nyt.csv`](https://github.com/covid19-data/covid19-data/blob/master/output/cases/cases_us_states_nyt.csv): US state-level historical case data from [New York Times](https://github.com/nytimes/covid-19-data)
+- [`owid-covid-data.json`](https://covid.ourworldindata.org/data/owid-covid-data.json): European Centre for Disease Prevention and Control (ECDC) historical world-wide case data (currently through [Our World in Data](https://ourworldindata.org/coronavirus-source-data)).
+
+- [`output/cases/cases_us_states_nyt.csv`](https://github.com/covid19-data/covid19-data/blob/master/output/cases/cases_us_states_nyt.csv): US state-level historical case data from [New York Times](https://github.com/nytimes/covid-19-data).
 
 ### Country metadata 
 
-- Country metadata: [`output/metadata/country/country_metadata.csv`](https://github.com/covid19-data/covid19-data/blob/master/output/metadata/country/country_metadata.csv) from [Worldbank](https://data.worldbank.org/indicator/SP.POP.TOTL).
-- ISO 3166-1 Alpha-3 country code conversion table. 
-    - [`output/metadata/country/country_name_code.csv`](https://github.com/yy/covid19-data/blob/master/output/metadata/country/country_name_code.csv): a conversion table from country name to code (ISO 3166 Alpha 3). Note that multiple names point to the same code.
-    - [`output/metadata/country/country_code_name.csv`](https://github.com/yy/covid19-data/blob/master/output/metadata/country/country_code_name.csv): a conversion table from country code (ISO 3166 Alpha 3) to country name. The shortest country names are picked from the above dataset.
+- World Bank Data: Classification of **world region** (Latin America & Caribben, South Asia, Sub-Saharan African, Europea & Central Asia, Middle East & North Africa, East Asia & Pacific, North America) for each country or area is based on [`data_source/metadata/worldbank/country_metadata.csv`](https://github.com/hongtaoh/covid19-data/blob/master/data_sources/metadata/worldbank/country_metadata.csv) from [World Bank](https://data.worldbank.org/indicator/SP.POP.TOTL).
+- ISO-3166-Countries-with-Regional-Codes: For countries or areas not found in World Bank data, their **world region** is found in [`ISO-3166-Countries-with-Regional-Codes`](https://github.com/hongtaoh/covid19-data/blob/master/data_sources/metadata/ISO-3166-Countries-with-Regional-Codes/all/all.csv).
+
 
 ### Historical case data for visualizations
 
-- [`cntry_stat_owid.json`](https://github.com/yy/covid19-data/blob/master/output/cntry_stat_owid.json): ECDC historical data merged with Worldbank's country metadata and ISO 3166-1 Alpha-3 data. Used in:
+- [`cntry_stat_owid.json`](https://github.com/hongtaoh/covid19-data/blob/master/output/cntry_stat_owid.json): ECDC historical data merged with Worldbank's classification of world regions. Used in:
```

Comment:
if we merge this, I think we want to have `covid19-data/covid19-data`

Reply:
That's right. I have just edited the `README.md`.